apps/communications/contacts/test/unit/views/details_test.js

```diff
@@ -153,6 +160,10 @@ suite('Render contact', function() {
     contacts.List = MockContactsListObj;
     realContacts = window.Contacts;
     window.Contacts = MockContacts;
+    realNFC = window.NFC;
+    window.NFC = MockContactsNfc;
```

Comment:
Why dont you use [1]?

[1] https://github.com/mozilla-b2g/gaia/pull/30669/files#diff-b0ddcc5353573a09c8cbf03040ff1663R101